http-server/apache2-web-server/httpd.def

```diff
@@ -1,9 +1,57 @@
 Bootstrap: docker
-From: httpd:latest
+From: httpd:2.4.37
+
+%help
+Apache2 web server running in a Singularity Container.
+For full tutorial, visit: https://github.com/sylabs/examples/tree/master/http-server/apache2-web-server
+
+usage:
+
+  setup:
+  $ mkdir -p httpd/logs
+  $ touch httpd/index.html  # this is your html file.
+
+  starting the instance:
+  $ singularity instance start -B httpd/:/srv/httpd/ httpd.sif httpd
```

Comment:
add the correct binding here as well: `$ singularity instance start -B ~/httpd/:/srv/httpd/ httpd.sif httpd`